src/Microsoft.ML.Transforms/Text/WordEmbeddingsTransform.cs

```diff
@@ -324,9 +343,93 @@ public Mapper(WordEmbeddingsTransform parent, Schema inputSchema)
                 _outputType = new VectorType(NumberType.R4, 3 * _parent._currentVocab.Dimension);
             }
 
+            public bool CanSaveOnnx(OnnxContext ctx) => true;
+
             public override Schema.Column[] GetOutputColumns()
                 => _parent.ColumnPairs.Select(x => new Schema.Column(x.output, _outputType, null)).ToArray();
 
+            public void SaveAsOnnx(OnnxContext ctx)
+            {
+                foreach (var (input, output) in _parent.Columns)
+                {
+                    var srcVariableName = ctx.GetVariableName(input);
+                    var schema = _parent.GetOutputSchema(InputSchema);
+                    schema.TryGetColumnIndex(output, out int colIndex);
+                    var dstVariableName = ctx.AddIntermediateVariable(schema.GetColumnType(colIndex), output);
+                    SaveAsOnnxCore(ctx, srcVariableName, dstVariableName);
+                }
+            }
+
+            private void SaveAsOnnxCore(OnnxContext ctx, string srcVariableName, string dstVariableName)
+            {
+                // Converts 1 column that is taken as input to the transform into one column of output
+                //
+                // Symbols:
+                // j: # of features per word in the pretrained model
+                // n: # of input words
+                // X: word input, n tensor
+                // k: # of words in pretrained model (known when transform is created)
+                // S: word labels, k tensor (known when transform is created)
+                // D: word embeddings, k-by-j tensor(known when transform is created)
+                //
+                // X[n]-- > LabelEncoder(classes_strings = S[k], default_int64 = 1)        D[k, j]->Initializer
```

Comment:
Please add a space between variable name and its dimension (also please check other places).
```suggestion
                // X [n]-- > LabelEncoder(classes_strings = S [k], default_int64 = 1)        D [k, j]->Initializer
```